modules/kubernetes/fluxcd-v2-azdo/templates/tenant.yaml

```diff
@@ -52,6 +53,24 @@ spec:
     name: ${name}
   prune: true
   validation: client
+%{ endif }
+%{ if include_tenant_name == true }
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: ${name}
+  namespace: ${name}
+spec:
+  serviceAccountName: flux
+  interval: 5m
+  path: ./tenant/${environment}/${name}
```

Comment:
I think that you could just put the if statement around the path instead of duplicating the Kustomization.

Reply:
Will change that